projects/04-training-api/requirements.md

```diff
@@ -0,0 +1,117 @@
+# Twitter API
+
+The purpose of the this project is to expose a GraphQL API.
+
+# Resources
+
+Go through the following before starting the implementation
+
+- [GraphQL Crash Course](https://www.youtube.com/watch?v=YFS5fAH2zFQ)
```

Comment:
Also recommend [this](https://www.youtube.com/watch?v=eIQh02xuVw4) & everything from that channel.